compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1651,9 +1653,18 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
             if (snippet != NULL)
                cg->stopUsingRegister(fakeTocRef->getModBase());
             }
+         else if (addr >= -0x200000000 && addr <= 0x1ffffffff)
```

Comment:
Better use macro constants which are used in elsewhere as well.

Reply:
Fixed in 7dd94a5eddd89d1164aebe3ea55fe879357ae5d9